src/index.js

```diff
@@ -2,11 +2,24 @@ const fs = require('fs');
 const path = require('path');
 const parser = require('./grammar.pegjs');
 
+const defaultOptions = {
+    filter: [],
+    invert: false
+};
+
+function validateOptions(options) {
+    if (!Array.isArray(options.filter) || typeof options.invert !== 'boolean')
+        throw new Error('Wrong type of option members!');
```

Comment:
Could we be a bit more specific here? Something along the lines of

```js
`Wrong argument type. Expected options.filter to be an array, but found ${typeof options.filter}`.
```

would make things much easier for users.

Reply:
Alternatively, instead of `found ${typeof options.filter}`, we could also use `found JSON.stringify(options.filter)}`, making it even more clear what got passed and wasn't, as expected, an array...

Reply:
Yeah, good idea.